free-programming-interactive-tutorials-en.md

```diff
@@ -282,6 +282,7 @@
 
 * [Intro to SQL: Querying and managing data](https://www.khanacademy.org/computing/computer-programming/sql) - Khan Academy
 * [SQL at Codecademy](https://www.codecademy.com/courses/learn-sql)
+* [SQL tutorial at w3schools](https://www.w3schools.com/sql)
```

Comment:
Please use the title as written on the resource.
If you want to denote it's from W3Schools, you can do so like all the other W3Schools links by appended it to the end as the vendor/author.

```suggestion
* [SQL Tutorial](https://www.w3schools.com/sql) - W3Schools
```